File: 1351-count-negative-numbers-in-a-sorted-matrix/1351-count-negative-numbers-in-a-sorted-matrix.cpp
class Solution {
public:
    int countNegatives(vector<vector<int>>& grid) {
        int n=grid.size();
        int m=grid[0].size();
        int ans=0;
        for(int i=0; i<n; i++){
            ans+=(grid[i].end() - lower_bound(grid[i].begin(), grid[i].end(), -1, greater<int>()));
        }
       return ans; 
    }
};

File: 1351-count-negative-numbers-in-a-sorted-matrix/1351-count-negative-numbers-in-a-sorted-matrix.java
class Solution {
    public int countNegatives(int[][] grid) {
        int count=0;
        for(int[] arr:grid){
           for(int i=0; i<arr.length; i++){
             if(arr[i]<0){
                 count+=arr.length-i;
                 break;
             }  
           }
        }
        return count;
    }
}